packages/charts/src/ChartFactory.js

```diff
@@ -48,12 +48,15 @@ const ChartFactory = React.memo(
       ...numberFormat
     },
     disableShowMore,
-    data,
+    data: propData,
     isComparison,
     comparisonData,
     referenceData,
     profiles
   }) => {
+    const data = useMemo(() => propData.map(d => ({ ...d, x: Number(d.x) })), [
```

Comment:
`text.split` in `computeMaxLabelDimension` expects this to be a `string` (default `Card` and `ChartFactory` stories fails).

Reply:
Yeah, looking through it now